pom.xml

```diff
@@ -40,4 +41,57 @@
 			</snapshots>
 		</repository>
 	</repositories>
+	
+	<profiles>
+		<profile>
+			<id>quality</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
```

Comment:
Why is it active by default ? What about dev running mvn on their laptop ? jacoco is already configured in the parent POM so why do we need it here ?